src/views/accounts/accounts.tsx

```diff
@@ -3,30 +3,34 @@ import {
   AddressText,
   Box,
   Breadcrumb,
+  Container,
   Flex,
-  PageContainer,
-  PageContainerProvider,
   SlideFade,
 } from "@liftedinit/ui"
 import { Layout } from "views"
 import { useGetAccountInfo } from "features/accounts"
 
 export function Accounts() {
   return (
-    <PageContainerProvider>
-      <Layout.Main>
-        <SlideFade in={true}>
-          <PageContainer>
-            <Box my={4}>
-              <AccountsHeader />
-            </Box>
-            <Box rounded="md" shadow="md" bgColor="white" p={4}>
-              <Outlet />
-            </Box>
-          </PageContainer>
-        </SlideFade>
-      </Layout.Main>
-    </PageContainerProvider>
+    <Layout.Main>
+      <SlideFade in={true}>
+        <Container
+          maxW={{
+            base: "auto",
+            md: "container.sm",
+            lg: "container.md",
+            xl: "container.lg",
+          }}
+        >
```

Comment:
Swapping in a Container for a PageContainer and adding the responsive sizes is the only change on this page. The rest is just formatting.